src/components/pages/PasswordReset/PasswordResetForm.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import uuid from 'uuid';
+
+import {
+  Container,
+  Button,
+  Grid,
+  Header,
+  Segment,
+  Form,
+  Image,
+  Message
+} from 'semantic-ui-react';
+import logo from '../../../assets/images/tiemendo_logo.jpg';
+import PageHeader from '../../common/PageHeader/PageHeader';
+
+const PasswordResetForm = props => {
+  const { state, handleSubmit, handleInputChange } = props;
+
+  return (
+    <>
+      <PageHeader />
```

Comment:
Can we put the `<PageHeader />` in the App.js so we don't have to put it inside every component (except Login)?